app/assets/styles/app.scss

```diff
@@ -180,7 +180,7 @@ h1.ui.header, h2.ui.header {
 }
 
 // Message
-.ui.positive.message {
+.ui.positive.message, .ui.success.message, .ui.error.message {
```

Comment:
yeah the green looked a bit weird with their color but I wasn't sure what to change it to... the teal of the app?